src/rules/unit-blacklist/README.md

```diff
@@ -0,0 +1,44 @@
+# unit-blacklist
+
+Specify a blacklist of disallowed units.
+
+```css
+    a { width: 100px; }
+/**               ↑
+ * These units */
```

Comment:
Very minor request, but can you indent `These units` so that the `s` is under the up arrow. It'll then match the style we use across the rules e.g. [here](https://github.com/stylelint/stylelint/blob/master/src/rules/function-comma-newline-after/README.md)